robottelo/hosts.py

```diff
@@ -1754,6 +1754,15 @@ def cli(self):
         self._cli._configured = True
         return self._cli
 
+    def install_satellite_or_capsule_package(self):
+        """Install Satellite/Capsule package.
+        Note: Make sure required repos are enabled before using this.
+        """
+        enable_module_cmd = f"dnf -y module enable {self.product_rpm_name}:el{settings.server.version.rhel_version} && "
+        if settings.server.version.rhel_version == 9:
+            enable_module_cmd = ""
+        return self.execute(f"{enable_module_cmd} dnf -y install {self.product_rpm_name}")
```

Comment:
Just to simplify and optimize this, and we can assert in same func if its installed instead of checking in tests
```suggestion
        if settings.server.version.rhel_version == 8:
            self.execute(f'dnf -y module enable {self.product_rpm_name}:el{settings.server.version.rhel_version}').status == 0
        assert self.execute(f'dnf -y install {self.product_rpm_name}').status == 0
```

Reply:
You beat me to it!

Reply:
updated:)